import/repos.json

```diff
@@ -87,7 +96,7 @@
     "repo": "https://github.com/EventStore/EventStoreDB-Client-Java",
     "branches": [
       {
-        "version": "5.3.0",
+        "version": "5.3.1",
```

Comment:
these versions will need to be updated in `docs/.vuepress/lib/samples.ts` as well or samples will break

Reply:
Thank you for catching that. I checked in an update that should address this.